lib/msal-browser/src/request/RedirectRequest.ts

```diff
@@ -46,4 +47,5 @@ export type RedirectRequest = Partial<
 > & {
     scopes: Array<string>;
     redirectStartPage?: string;
+    navigateToLoginRequestUrl?: boolean;
```

Comment:
It feels wrong for this to be part of `RedirectRequest`, what needs to happen to make this only applicable to `handleRedirectPromise?